server/ecore-glsp/pom.xml

```diff
@@ -178,6 +178,24 @@
 					</execution>
 				</executions>
 			</plugin>
+	        <plugin>
+	            <groupId>org.apache.maven.plugins</groupId>
+	            <artifactId>maven-antrun-plugin</artifactId>
+	            <version>1.8</version>
+	            <executions>
+	                <execution>
+	                    <phase>install</phase>
```

Comment:
wouldn't it be better to bind this to the `verify` phase? This way the package would not be installed into the local mvn repository

Reply:
In general I would have assumed, that the developer runs `mvn install ` anyways. Changing the phase for the plugin wouldn't prevent an installation in local repo in that case.

But of course it can be bound differently.

Reply:
I prefer to run `mvn clean verify` because of the local install otherwise ;-)

Reply:
+1 for <= `verify`; installing is only useful if you intend to do partial builds, so it shouldn't be the default phase.

I think `package` would even be the right phase, as `verify` is the integration test phase

Reply:
Is now part of `package` phase